pilot/cmd/pilot-agent/main.go

```diff
@@ -144,6 +144,7 @@ var (
 	useTokenForCSREnv   = env.RegisterBoolVar("USE_TOKEN_FOR_CSR", false, "CSR requires a token").Get()
 	credFetcherTypeEnv  = env.RegisterStringVar("CREDENTIAL_FETCHER_TYPE", "",
 		"The type of the credential fetcher. Currently supported types include GoogleComputeEngine").Get()
+	parseTokenEnv = env.RegisterBoolVar("PARSE_TOKEN", false, "Parse token to inspect information like expiration time").Get()
```

Comment:
Can you explain (in the code as a comment/description on the env) *why* we do not parse the token?

Reply:
Because the token may not always be a JWT, proxy may not be able to parse token. And proxy probably SHOULD not parse token unless explicitly required. In fact today, we skip parsing token (i.e., AlwaysValidTokenFlag is true) if PROV_CERT is set. I will update the comments.